src/i18n/en.js

```diff
@@ -432,17 +432,17 @@ Want to know more? Press [HELP](${urlHelp}/contact-policy).`,
       has_campaings_created_description_MD: `
 You can choose between three Campaigns types according to your business goals. [Get down to work!](${urlDopplerLegacy}/Campaigns/BasicInfo)
 Want to go further? Try **Automatized Email and SMS Flows!**`,
-      has_campaings_created_title: 'Create your first Campaign',
+      has_campaings_created_title: `#### [Create your first Campaign](${urlDopplerLegacy}/Campaigns/BasicInfo)`,
       has_campaings_sent_description_MD: `
 Analyze its results is essential to take decisions and optimize your strategy. [Find out the Doppler Reports section](${urlSentCampaigns}) or [find out the meaning of each indicator](${urlSummaryReport}).
       `,
-      has_campaings_sent_title: 'Look at the metrics of your Campaign',
+      has_campaings_sent_title: `#### [Look at the metrics of your Campaign](${urlSentCampaigns})`,
```

Comment:
No se si es buena idea poner el h4 como parte del contenido.

Osea, si unieras título y detalle en la misma entrada, creo que si tendría sentido, pero en este caso me suena que no.

Por ejemplo, supongamos que surge la necesidad de modificar el h4 por h5, eso seguramente ocurra en el contexto de modificaciones en `FirstSteps/ActionBox/index.js`, no en modificaciones de idioma.

Reply:
Por otro lado, es raro que un título tenga link, en todo caso debería ser un link a si mismo. Estás seguro de que esto es un título?

Reply:
Realmente no lo veo mal, igual forma parte de la especificación de markdown ... 
Por otro lado, inicialmente pensé en dejar el `h4` fuera del archivo de traducciones, pero me enfrento a construir esta estructura: 

`<h4><a href="#">Crea una lista y añade Contactos</a></h4>`

Intenté cambiar el nodo wrapper que me genera el markdown, pero no tuve éxito. Lo que me genera es esto: 

`<h4><p><a href="#">Crea una lista y añade Contactos</a></p></h4>`

Reply:
Si se te ocurre algo mejor, bienvenido 🙏

Reply:
Inicialmente tenían pensado hacer que toda la caja fuera un link, por lo que entiendo quieren resaltar el enlace que está en cada card, pero luego Gus comentaba que esto era semanticamente incorrecto por los elementos en bloque y en linea. Resultaba mejor haciendo que solo el titulo tuviera el enlace. @GusBaamonde me corriges si me equivoco

Reply:
> Realmente no lo veo mal, igual forma parte de la especificación de markdown

Claro, pero así y todo creo que no encaja. @silsanchez8 sintió algo parecido aquí: https://github.com/FromDoppler/doppler-webapp/pull/1787/files#r766751569

Respecto a que te agruega el `<p>`, creo que se podría hacer algo así:

```
        <FormattedMessageMarkdown id={titleId} wrapper-element="h4" />
```

De esa forma, no tenés el `<p>` y el elemento usado queda del lado del componente.

Reply:
Alli hago la prueba @andresmoschini

Reply:
Lo probé así `<FormattedMessageMarkdown id={titleId} wrapper-element="h4" />`, pero esto es lo que me genera:

<img width="404" alt="Captura de Pantalla 2021-12-10 a la(s) 10 30 42 a  m" src="https://user-images.githubusercontent.com/84402180/145599277-d4740e1d-b2b6-46ae-b642-e8fa52e735a8.png">

Reply:
I was confounding it with the property `tagName` of `FormattedMessage`. We are using this here: https://github.com/FromDoppler/doppler-webapp/blob/2cfe6676b3fa0169c4b284a4dfa59a3071e62cbe/src/components/SiteTrackingRequired/SiteTrackingRequired.js#L72

But here we are not using `FormattedMessage`.

Sorry for the noise.